server/controllers/reservationController.js

```diff
@@ -72,8 +71,19 @@ const testAllReservations = async (garageId) => {
   }
 }
 
-const createReservation = () => {
+const createReservation = async (reservation) => {
+  try {
+    const { data, error } = await Reservations
+      .insert(reservation)
```

Comment:
looking back here after I read the reservations router code, would strongly suggest for future readability a comment explaining what properties are within this reservation object.

Reply:
Will add these requirements in the README file